js/ui/scheduler/workspaces/ui.scheduler.work_space.js

```diff
@@ -2037,13 +2037,15 @@ class SchedulerWorkSpace extends WidgetObserver {
         if(!this.dragBehavior && scheduler) {
             this.dragBehavior = new AppointmentDragBehavior(scheduler);
 
-            this._createDragBehavior(this.getWorkArea());
-            this._createDragBehavior(this.getAllDayContainer());
-            this._createDragBehavior(this._$allDayPanel);
+            const element = $(scheduler.element());
```

Comment:
Following our code style, do we need to add the $ symbol to jQuery instances?
In my point of view, we need to add the $ symbol :)
Like this:
```js
const $element = $(scheduler.element());
```